src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -359,6 +359,10 @@
 	border-color: var(--vscode-focusBorder);
 }
 
+.interactive-session .interactive-input-and-execute-toolbar .monaco-editor .mtk1 {
```

Comment:
Isn't mtk1 an internal concept to monaco? I was assuming these could change at any time.

Reply:
Yes. But there is no other way to change the text color from the outside, and we already do this for the other editors that are used as inputs. I talked to Peng about it. I'll open an issue to get an official way to do this in the editor